### PR TITLE
getdns: init -> 1.3.0

### PR DESCRIPTION
**Pull request by leenaars**
###### Motivation for this change

This is a very useful library that provides an abstraction of DNS to application developers.
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/libraries/getdns/default.nix [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  version = "1.3.0";
+
+  src = fetchurl {
+    url = "https://getdnsapi.net/releases/${pname}-1-3-0/${pname}-${version}.tar.gz";
+    sha256 = "920fa2e07c72fd0e5854db1820fa777108009fc5cb702f9aa5155ef58b12adb1";
+  };
+
+  nativeBuildInputs = [ libtool m4 libbsd ];
+
+  buildInputs = [ unbound libidn openssl doxygen autoreconfHook
```
Comment (adisbladis) [inline pkgs/development/libraries/getdns/default.nix]: `autoreconfHook`, `automake` and `file` also belongs in `nativeBuildInputs`.

Comment (rasendubi) [reply pkgs/development/libraries/getdns/default.nix]: I believe `doxygen` is a build-time only dependency as well.

File: pkgs/development/libraries/getdns/default.nix [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+   automake file check ];
+
+  patchPhase = ''substituteInPlace m4/acx_openssl.m4 \
```
Comment (rasendubi) [inline pkgs/development/libraries/getdns/default.nix]: Format it like this:
```
patchPhase = ''
  substituteInPlace m4/acx_openssl.m4 \
    --replace /usr/local/ssl ${openssl.dev}
'';
```

File: pkgs/development/libraries/getdns/default.nix [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    --replace /usr/local/ssl ${openssl.dev}
+    '';
+
+  meta = with stdenv.lib; {
+    description = "A modern asynchronous DNS API";
+    longDescription = ''
+getdns is an implementation of a modern asynchronous DNS API; the
```
Comment (rasendubi) [inline pkgs/development/libraries/getdns/default.nix]: You can safely indent the description inside `''` quote.

File: pkgs/development/libraries/getdns/default.nix [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+make all types of DNS information easily available to application
+developers and non-DNS experts. DNSSEC offers a unique global
+infrastructure for establishing and enhancing cryptographic trust
+relations. With the development of this API the developers intend to
+offer application developers a modern and flexible interface that
+enables end-to-end trust in the DNS architecture, and which will
+inspire application developers to implement innovative security
+solutions in their applications.
+'';
+    homepage = "https://getdnsapi.net";
```
Comment (rasendubi) [inline pkgs/development/libraries/getdns/default.nix]: Use
```
homepage = https://getdnsapi.net;
```

(URL is a first-class type in Nix.)

Comment (leenaars) [reply pkgs/development/libraries/getdns/default.nix]: Done.

**Comment by leenaars** [comment]
All comments processed.

**Comment by adisbladis** [comment]
@leenaars I dont think you meant to close this?

**Comment by vcunat** [comment]
GitHub PRs are updated via (force-)pushing the branch from which they're filed.

**Comment by leenaars** [comment]
@adisbladis: I thought someone merged (it says: "Pull request successfully merged and closed"). But this seems an error @github, as my PR has vanished. I'll submit another pull request.

**Pull request closed without merging**